content/en/docs/reference/node/kernel-version-requirements.md

```diff
@@ -0,0 +1,59 @@
+---
+content_type: "reference"
+title: OS Kernel Version Requirements
+weight: 10
+---
+
+Many features rely on specific kernel functionalities and have minimum kernel version requirements.
```

Comment:
Please add the third party software disclaimer using a [shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#third-party-content-marker); things mentioned here are `runc` and Linux.